src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    return totalCost - 50;
+  }
+
+  if (days >= 3) {
+    return totalCost - 20;
+  }
```

Comment:
Hey Swazi, I don't necessarily agree that else if statements === clean code. I was taught before to avoid them if it is possible. They for sure make the code more compact, but at the same time not very readable, especially if there are multiple.

Reply:
agree with Lukasz

Reply:
but let's give 20 and 50 some context. i have no idea what they are when i look at this code